src/pages/Search/Search.tsx

```diff
@@ -44,13 +41,11 @@ const Search = () => {
     }
   }, [firstRender, query, searchQuery, updateSearchQuery]);
 
-  const onCardClick = (playlistItem: PlaylistItem) => {
+  const onCardClick = () => {
     useUIStore.setState({
```

Comment:
Maybe it's cleaner to just do this in on-unmount now that we aren't navigating via card clicks.
https://legacy.reactjs.org/docs/hooks-effect.html#effects-with-cleanup
```suggestion
  useEffect(() => {
    return () => {
      useUIStore.setState({
        searchQuery: '',
        searchActive: false,
      });
    }
  }, []);
```